activiti-cloud-acceptance-tests/activiti-cloud-acceptance-tests-core/src/main/java/org/activiti/cloud/acc/core/services/runtime/ProcessVariablesRuntimeService.java

```diff
@@ -29,7 +29,7 @@ public interface ProcessVariablesRuntimeService {
     @Headers("Accept: application/hal+json;charset=UTF-8")
     CollectionModel<CloudVariableInstance> getVariables(@Param("id") String id);
 
-    @RequestLine("POST /v1/process-instances/{id}/variables")
+    @RequestLine("PUT /v1/process-instances/{id}/variables")
     @Headers("Content-Type: application/json")
     ResponseEntity<Void> setVariables(@Param("id") String id,
```

Comment:
Shouldn't this method also be renamed?

Reply:
I didn't rename it because it is used by the acceptance tests in `activiti-cloud-application` and its name seemed coherent with the name of the get method.

Reply:
Done